examples/snn/run.sh

```diff
@@ -17,35 +17,69 @@ do
 done
 
 EXECUTION=$exe_file
+DEVICE=1
+
+mkdir JSON
 
 # addresses
-DATADIR="/data/gunrock_dataset/large"
+PATH[0]="/data/clustering_dataset/KDDCUP04Bio"
+PATH[1]="/data/clustering_dataset/3DRN"
+PATH[2]="/home/agnes/clustering_dataset/Font/dim11"
+PATH[3]="/home/agnes/clustering_dataset/Halo/dim9"
+PATH[4]="/home/agnes/clustering_dataset/Halo/dim9"
+PATH[5]="/home/agnes/clustering_dataset/Font/dim11"
+PATH[6]="/home/agnes/clustering_dataset/Berton/dim3"
+PATH[7]="/home/agnes/clustering_dataset/Berton/dim3"
+PATH[8]="/home/agnes/clustering_dataset/Bower/dim3"
+PATH[9]="/home/agnes/clustering_dataset/FOF"
+PATH[10]="/home/agnes/clustering_dataset/FOF"
+PATH[11]="/home/agnes/clustering_dataset/FOF"
+PATH[12]="/home/agnes/clustering_dataset/FOF"
+PATH[13]="/home/agnes/clustering_dataset/FOF"
+PATH[14]="/home/agnes/clustering_dataset/FOF"
+PATH[15]="/home/agnes/clustering_dataset/FOF"
```

Comment:
Let's update these paths to `/data/clustering_dataset/` and I will add all the other datasets to that main path.

Reply:
I am going to remove this file. Those paths does not exists anymore